### PR TITLE
DOC: Improve docstrings for readthedocs

### PR DESCRIPTION
**Pull request by simongraham**
Cleaned up docstrings so that the output of readthedocs was correct.

For this, it was important to use 2 spaces, rather than an indent with 4 spaces when going to a continuing text onto a new line in the docstring.

The `rst` file for semantic segmentation is not yet done - this needs to be completed.

- Some docstrings have a colon. For example, https://github.com/TIA-Lab/tiatoolbox/blob/develop/tiatoolbox/tools/patchextraction.py#L42 - this needs to be addressed otherwise readthedocs will have an undesirable output.

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -128,6 +128,25 @@ Classification
 .. automodule:: tiatoolbox.models.classification.pretrained_info
     :members:
 
+------------
```
Comment (shaneahmed) [inline docs/usage.rst:131]: Is it possible to decouple this PR from segmentation models? To help with the release with patch predictor?

You can add these changes to semantic segmentation PR.

File: tests/test_models_semantic_segmentation.py [hunk 2]
```diff
@@ -0,0 +1,451 @@
+import copy
```
Comment (shaneahmed) [inline tests/test_models_semantic_segmentation.py:1]: Same as above, if we can add these changes to the semantic segmentation that would be great. Just to help with the release on patch predictor.

File: tiatoolbox/models/segmentation/__init__.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# ***** BEGIN GPL LICENSE BLOCK *****
```
Comment (shaneahmed) [inline tiatoolbox/models/segmentation/__init__.py:1]: same as above

File: tiatoolbox/models/segmentation/semantic_segmentor.py [hunk 4]
```diff
@@ -0,0 +1,984 @@
+# ***** BEGIN GPL LICENSE BLOCK *****
```
Comment (shaneahmed) [inline tiatoolbox/models/segmentation/semantic_segmentor.py:1]: same as above

**Review by shaneahmed — COMMENTED** [review]
Thanks @simongraham 
Overall it looks fine to me. Would it be possible to decouple this PR from the segmentation PR? Just to help with the release? You can merge the changes related to segmentation branch on its PR.

File: tiatoolbox/models/classification/patch_predictor.py [hunk 5]
```diff
@@ -152,11 +152,9 @@ class CNNPatchPredictor:
         >>> data = [img1, img2]
```
Comment (shaneahmed) [inline tiatoolbox/models/classification/patch_predictor.py:152]: The Attributes list is not complete. I think you are missing `pretrained_models` and `pretrained_weights`.

File: tiatoolbox/models/classification/patch_predictor.py [hunk 6]
```diff
@@ -152,11 +152,9 @@ class CNNPatchPredictor:
...
         >>> predictor = CNNPatchPredictor(pretrained_model="resnet18-kather100k")
         >>> output = predictor.predict(data, mode='patch')
-
         >>> data = np.array([img1, img2])
         >>> predictor = CNNPatchPredictor(pretrained_model="resnet18-kather100k")
         >>> output = predictor.predict(data, mode='patch')
-
         >>> wsi_file = 'path/wsi.svs'
```
Comment (shaneahmed) [inline tiatoolbox/models/classification/patch_predictor.py:158]: Add an example with all the available options for `CNNPatchPredictor`.

Comment (simongraham) [reply tiatoolbox/models/classification/patch_predictor.py]: Addressed

**Review by shaneahmed — APPROVED** [review]
Thanks @simongraham Readthedocs look fine to me now.

**Pull request merged**